zulipterminal/themes/gruvbox.py

```diff
@@ -22,13 +24,7 @@ class GruvBoxColor(Enum):
     FADED_RED        = 'dark_red        h88    #9d0006'
 
 
-Color = Enum(
-    "Color",
-    {
-        **{c.name: c.value for c in GruvBoxColor},
-        **{c.name + "__BOLD": c.value + " , bold" for c in GruvBoxColor},
-    },
-)
+Color = add_color_prop(GruvBoxColor, 'BOLD')
```

Comment:
I first generate the files in the first few commits and then write the code to use the in later commits. I can't have `helper.py` first because I need a theme to test it with.
I can remove the reference though.

Reply:
How about `color_properties(GruvBoxColor, 'BOLD')` ?

Reply:
Re cyclic import, the issue is themes.py imports all the theme files. If `color_properties` were in themes.py then:
`themes.py -> template/gruvbox -> themes.py`
Error

But now that is in helper.py:
`themes.py -> template/gruvbox -> helper.py`
No problem, though type hinting caused another cyclic import which was solved using TYPE_CHECKING.
If it were in themes.py I would have 2 cyclic import problems.

Reply:
Actually I think I can move the helper behind.

Reply:
Maybe `color_with_extra_properties`?

Reply:
extra would mean there were already some properties?
`color_with_added_properties`?

Reply:
Is the current name `color_properties` fine or is any change needed?

Another thing is that in the current implementation of color_properties doing something like
color_properties(Color, BOLD, ITALICS) would result in only Color.WHITE and Color.WHITE__BOLD_ITALICS but
not Color.WHITE__BOLD or Color.WHITE__ITALICS. One would also have to do
color_properties(Color, BOLD) and color_properties(Color, ITALICS) for the others to work.

I think the current implementation is good for now as other properties are not used and this doesn't restrict its use aswell. Is a FIXME or NOTE enough?